pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/instance/DDPInstanceDao.java

```diff
@@ -63,6 +64,10 @@ public class DDPInstanceDao implements Dao<DDPInstanceDto> {
                     + "AND realm.ddp_instance_id = main.ddp_instance_id) AS 'has_role', mr_attention_flag_d, tissue_attention_flag_d, "
                     + "auth0_token, notification_recipients FROM ddp_instance main "
                     + "WHERE is_active = 1";
+    private static final String SQL_SELECT_ROLES_FOR_INSTANCE =
```

Comment:
if that query is not used anywhere else, please add the by_instance_name part as well